aiocache/decorators.py

```diff
@@ -183,12 +183,20 @@ def __init__(self, lease=2, **kwargs):
         super().__init__(**kwargs)
         self.lease = lease
 
-    async def decorator(self, f, *args, **kwargs):
+    async def decorator(self,
+        f,
+        *args,
+        cache_read=True,
+        cache_write=True,
+        aiocache_wait_for_write=True,
```

Comment:
these changes seem unrelated with what you propose, can you remove them please?

Reply:
Agreed. The cache_read and cache_write should be a new issue and a different MR if you'd like to propose that functionality. It's a different discussion (feature add) as opposed to the original bug fix.